source/systems/common.rst

```diff
@@ -64,5 +64,1789 @@ RDHPCS Platforms
+to customize these shell scripts, we have seen a large number of issues related
+to loading modules within these shell scripts.
+
+Loading modules in shell initialization scripts can lead to unintended
+consequences, as the shell's environment may be different than the one
+expected. The wrong libraries can be loaded, the wrong tools can be used, the
+wrong version of tools can be used, and even tools provided with the operating
+system may no longer work properly or provide strange error messages. For these
+reasons, we **highly** recommend that you do not add module loads to your
+shell's initialization scripts.
+
+Instead, we recommend that you remove module loads from shell initialization
+scripts and do one or more of the following:
+
+#. Add the module loads directly to your batch script or cron scripts
+#. Create a separate script responsible for loading the desired modules and
+   environment. This script can then be invoked/sourced any time you want to
+   set up this specific environment. A command "alias" can also be added to
+   your shell's initialization scripts. You can then run the alias command to
+   invoke the desired shell environment.
+#. Create a script as described above and have all members of your project
+   invoke/source the exact same script. This will ensure that the exact same
+   modules are used by all users. You can even add "module purge" to the
+   beginning of the script to ensure that only the desired modules are being
+   loaded.
+
+
+If you need help implementing these methods, open an RDHPCS help ticket. See
+:ref:`Getting_Help` for details.
+
+Job Submission Options
+----------------------
+
+sbatch options
+^^^^^^^^^^^^^^
+
+You are allowed to specify options from teh set used for the
+SLURM batch system. For a list of options, you may look at the man page:
+
+``$ man sbatch``
+
+or the command usage statement:
+
+``$ sbatch --help``
+
+Additional sbatch information can be found at the `vendor's website <https://slurm.schedmd.com/sbatch.html>`_.
+
+Command-line options vs directive options
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are two way to specify sbatch options. The first is on the command line
+when issuing the sbatch command. For example,
+
+``$ sbatch -A fim --ntasks=256 jobscript.sh``
+
+The second method is to insert directives at the top of the batch script using
+#SBATCH syntax. For example:
+
+
+.. code-block:: shell
+
+   #!/bin/bash -l
+
+   #SBATCH -A fim
+   #SBATCH --ntasks=256
+
+The two methods may be mixed together, if desired. Options specified on the
+command line always override options specified in the script.
+
+A Quick Start Batch Script Example
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This script is a very basic template that provides examples for some common
+sbatch options. It also includes the required options. This can be
+used as a general guide when constructing a new batch script.
+
+.. code-block:: shell
```

Comment:
This script should be simplified so it can run on ALL RDHPCS systems.  Users will not have access to the `xhpl` command.